pykern/pkcli/argh_test.py

```diff
@@ -0,0 +1,25 @@
+"""Test pkcli to handle argh breaking changes
```

Comment:
This should be unit test data. See below.

Reply:
1afbd34